src/legacy/core_plugins/kibana/public/discover/doc_table/_doc_table_source_mixin.scss

```
@@ -0,0 +1,27 @@
+/**
+*  Style ES document _source in table view <dt>key:<dt><dd>value</dd>
+*  Use alpha so this will stand out against non-white backgrounds, e.g. the highlighted
+*  row in the Context Log.
+*/
+@mixin style_doc_source {
```

Lets be specific with the naming of mixins too, including the prefix (even though it's used outside of discover, discover is the one creating it so we will know where it's coming from).
```suggestion
@mixin dscDocSourceStyle {
```

---

definitly ... done